device/mellanox/x86_64-mlnx_msn4700_simx-r0/platform_reboot

```diff
@@ -0,0 +1 @@
+../x86_64-mlnx_msn4700-r0/platform_reboot
```

Comment:
need to fix to point to a simx reboot file not to real hw system

Reply:
chnaged